### PR TITLE
Loginpage - View updated!

### PR DESCRIPTION
**Pull request by Yaamanni**

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -33,7 +33,7 @@
   </div>
 
   <div class="form-actions">
-    <%= f.button :submit, "Update" %>
+    <%= f.button :submit, "Update", , class:"btn-medium"  %>
```
Comment (prawnto) [inline app/views/devise/registrations/edit.html.erb:36]: could you run rubocop? might be an offense / typo for the two , ,

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
- <h2>Sign up</h2>
+<h2>Sign up</h2>
 
-
-  <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name),
+<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name),
   data: {turbo: false}) do |f| %>
-
   <div class="form-inputs">
+    <%= f.input :first_name,
```
Comment (prawnto) [inline app/views/devise/registrations/new.html.erb:6]: is last_name intentionally omitted from registration?

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
+                required: false,
```
Comment (prawnto) [inline app/views/devise/registrations/new.html.erb]: might be good to make the names a requirement:

required: true,

File: app/views/devise/sessions/new.html.erb [hunk 4]
```diff
@@ -3,7 +3,6 @@
 <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name),
   data: {turbo: false}) do |f| %>
   <div class="form-inputs">
-
     <%= f.input :email,
                 required: false,
```
Comment (prawnto) [inline app/views/devise/sessions/new.html.erb:7]: will the users be able to sign in if required is defined as false?

**Review by prawnto — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/components/_buttons.scss [hunk 5]
```diff
@@ -3,10 +3,31 @@
...
   0%		{ background-color: $grab; }
   25%		{ background-color: $grab-secondary; }
   50%		{ background-color: $grab; }
   75%		{ background-color: $grab-secondary; }
   100%	{ background-color: $grab; }
 }
+
+.btn-medium {
+	background-color: #1e1c1c;
+  color: #ffffff;
```
Comment (goh0222) [inline app/assets/stylesheets/components/_buttons.scss:17]: fix indentation

File: config/routes.rb [hunk 6]
```diff
@@ -1,13 +1,12 @@
...
+  get 'wishlist/new'
 
-  resources :bookings, only: [:index, :destroy]
+  root "page#index"
 
-  resources :reviews
+  resources :listings
+  resources :bookings
+  resources :reviews, only: [:create, :index]
+  resources :wishlist, only: [:create, :index]
```
Comment (goh0222) [inline config/routes.rb:10]: @Yaamanni your changes are overwriting the listings and bookings stuff

**Pull request closed without merging**